src/berry_mill/kiwrap.py

```diff
@@ -145,7 +166,7 @@ def _key_selection(self, reponame: str, options: List[str]) -> str | None:
                           ),
         ]
         answer = inquirer.prompt(question)
-        print("You selected:", answer["choice"])
+        log.info("You selected:", answer["choice"])
```

Comment:
please leave it at print().

using the logging facility at this point (while the user is prompted for input) will cause issues.